htdocs/info/buildings.php

```diff
@@ -417,10 +432,9 @@
     }
 
     echo "<input type=\"button\" value=\"Geb&auml;ude&uuml;bersicht\" onclick=\"document.location='?$link&amp;site=$site'\" /> &nbsp; ";
-    if (!$popup) {
```

Comment:
The popup is still used by the discovery design. Should we remove it there too and fully get rid of the popups inside the game too? Currently the `Technikbaum` link doesn't really work.

Reply:
I think I would vote for removing it.